HoloPipelines/pipelines/components/compDcm2nifti.py

```diff
@@ -9,7 +11,9 @@ def main(dicomInputPath, outputNiftiFolderPath):
     # convert series of dicom to nifti
     dicom2nifti.convert_directory(dicomInputPath, outputNiftiFolderPath)
     print("dcm2nifti: done")
-    return outputNiftiFolderPath
+    lsdir = glob.glob(str(pathlib.Path(outputNiftiFolderPath).joinpath("*.nii.gz")))
```

Comment:
@UdomkarnBoonyaprasert  You changed the return to the filepath of the first `nii.gz` file found instead of the folder. Won't this affect the pipeline that uses this component? 

Do explain the change here.

Reply:
In short, no. But I think I should revisit the pipeline/components which accepts both full path and a folder path of `nii.gz`, and make it so that it accepts the full path to `.nii.gz`

I made the change to `compDcm2nifti` as I think it would make more sense to return the full path from the component itself and not having pipeline to figure out the `.nii.gz` from a folder path.

Reply:
Ok i just understood why it still works. The lung segmentation module assumes the input argument (inputNiftiPath) could either be a directory or a file. 

But even so, this is also not a change within the scope of this PR.

Reply:
Yes, I do apologizes about that.

Reply:
Thanks for resolving this issue. Just for the record: When you continue working on this on another branch, please try to have modular code, as discussed. I.e. ideally, each component should have the same interface, same I/O. I like "dumb" components that have strict constraints what their I/O is over smart components that figure out different input specs depending on the context.

Reply:
I have pushed a commit just to undo the changes here. This standardisation of I/O of components should come from its own PR.